### PR TITLE
Transactions component

### LINKED ISSUE #315: Add styles for card
Add styling for card element

### PR DESCRIPTION
**Pull request by dejvid15**
Closes #315

### CODE REVIEW

File: app/public/css/styles.css [hunk 1]
```diff
@@ -1477,3 +1476,65 @@ p {
...
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
+}
+
+/* TRANSACTIONS */
+
+.transactions {
+  padding: 40px 56px;
+}
+
+.transactions-title {
```
Comment (darjanin) [inline app/public/css/styles.css]: This is the same for all `.card`s.

Please, rename it to `.card-title` and move it to CARD.

File: app/public/css/styles.css [hunk 2]
```diff
@@ -1477,3 +1476,65 @@ p {
...
+
+.card {
+  background-color: white;
+  padding: 40px 56px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
+}
+
+/* TRANSACTIONS */
+
+.transactions {
```
Comment (darjanin) [inline app/public/css/styles.css]: This is redundant to line 1484. Please remove it.

File: app/public/css/styles.css [hunk 3]
```diff
@@ -1477,3 +1476,65 @@ p {
...
+  display: grid;
+  grid-template-columns: 50% 50%;
+  padding-top: 24px;
+  padding-bottom: 24px;
+  border-top: 1px solid var(--color-border);
+  color: var(--color-grey);
+}
+
+.transaction-date {
+  display: flex;
```
Comment (darjanin) [inline app/public/css/styles.css]: This is not the proper way how to do it if you use the *grid*. The grid supports `align-self: center;` for vertical alignment in a grid item.

File: app/frontend/components/pages/txHistory/transactionHistory.js [hunk 4]
```diff
@@ -1,106 +1,72 @@
...
-        h(
-          'thead',
-          undefined,
+    {class: 'transactions card'},
+    h('h2', {class: 'transactions-title'}, 'Transaction History'),
+    /* TODO: Load transaction items by chunks */
+    transactionHistory.length === 0
+      ? h('div', {class: 'transactions-empty'}, 'No transactions found')
+      : h(
+        'div',
```
Comment (darjanin) [inline app/frontend/components/pages/txHistory/transactionHistory.js]: This is list of transactions. It should be in `ul`.

File: app/frontend/components/pages/txHistory/transactionHistory.js [hunk 5]
```diff
@@ -1,106 +1,72 @@
...
-            ),
-            h(
-              'th',
-              undefined,
-              'Fee (',
-              h('img', {src: 'assets/ada-icon.svg', className: 'ada-sign-inline'}),
-              ')'
-            ),
-            h('th', undefined, 'Transaction')
+            'div',
```
Comment (darjanin) [inline app/frontend/components/pages/txHistory/transactionHistory.js]: `li`.

File: app/public/css/styles.css [hunk 6]
```diff
@@ -1477,3 +1476,65 @@ p {
...
+
+.transaction-amount.credit {
+  color: var(--color-success);
+}
+
+.transaction-amount.debit {
+  color: var(--color-error);
+}
+
+.transaction-address {
```
Comment (darjanin) [inline app/public/css/styles.css]: Please fix this issue. You can misclick _View details_.
screen recording 2019-01-15 at 12 48 05 2019-01-15 12_59_12

**Comment by dejvid15** [comment]
@darjanin Implemented your review comments

**Pull request merged**